core/org.wso2.carbon.bootstrap/src/main/java/org/wso2/carbon/bootstrap/CheckinClientBootstrap.java

```diff
@@ -76,6 +76,13 @@ protected void addClassPathEntries() throws MalformedURLException {
 		addFileUrl(new File(ROOT + File.separator + LIB + File.separator
 				+ "checkin-client" + File.separator + "log4j.properties"));
 
+		//add component/plugins
+		String internalLibPath = System.getProperty("components.repo");
```

Comment:
I have moved most of them. In this class, we don't have access to inner carbon class path.